tests/cypress/support/commands.js

```diff
@@ -88,28 +88,55 @@ Cypress.Commands.add('openTaskJob', (taskName, jobNumber=0) => {
     cy.openJob(jobNumber)
 })
 
-Cypress.Commands.add('createShape', (firstX, firstY, lastX, lastY) => {
+Cypress.Commands.add('createShape', (firstX, firstY, secondX, secondY, points='By 2 Points',
```

Comment:
Can we sort these arguments? It's bad practice to mix argument with/without default value. I would suggest to pass here an object.

Reply:
Reworked to use the object as a parameter. Renamed the function for clarity. In this regard, I adapted tests that use this function.